playground/nextjs/pages/index.tsx

```diff
@@ -71,6 +71,14 @@ export default function Home() {
 
                 <button className="ph-no-capture">Ignore certain elements</button>
 
+                <button
+                    onClick={() => {
+                        posthog?.reloadFeatureFlags()
```

Comment:
I just included this button here as it's a reliable way to trigger a call to the flags endpoint